Building.py

```diff
@@ -0,0 +1,42 @@
+import json
+import requests
+
+
+class Building:
+    def __init__(self, id, name, address, location, images=[], directional_check=False):
+        self.id = id
+        self.name = name
+        self.address = address
+        self.location = location
+        self.images = images
+        self.directional_check = directional_check
+
+    def __str__(self):
+        return '<Building id=%s>' % self.id
+
+    def __repr__(self):
+        return json.dumps(self.__dict__)
+
+    @staticmethod
+    def get_by_position(lon, lat):
+        response = []
+        headers = []
+        try:
+            request = requests.get("localhost", params=headers)
```

Comment:
That hardcoded URL would better be set via the environment as well.

Also, where are the docs? How is Twitter.py and Building.py relate to each other?